.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,18 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Build the Docker image
+      run: docker build . --file Dockerfile --tag my-image-name:$(date +%s)
```

Comment:
That tag seems wrong, should probably be based on like a docker repository that Github might provide, nah?

cf. https://git.pleroma.social/pleroma/pleroma/-/blob/develop/ci/build_and_push.sh

Reply:
feel free to improve